### PR TITLE
Added Test Setup

### LINKED ISSUE #18: Add testing notes to ReadMe
In the readme file, after the How to Setup and before the How to view api section - Add the testing section. 
I have included the format for the testing section below: 

## How to run Test ##
To run initial database seeder: `php artisan db:seed`
To remove tables and add new data: `php artisan migrate:fresh --seed`
If you make changes run the command again (php artisan db:seed) to confirm changes in database

### PR DESCRIPTION
**Pull request by LuluNwenyi**
I've added the instructions to test the program.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,7 @@ If you would like to follow along in the series - The playlist link for this vid
 Use Postman [install here](https://www.postman.com/downloads/) or browser to visit api route
     - Example route to visit [Get all users](http://appointment.test/api/users)
 
+## How to run Test
+- Initialise the database seeder by running the command: `php artisan db:seed`
+- Remove the tables and add new data: `php artisan migrate:fresh --seed`
```
Comment (tthompson899) [inline README.md]: Request: change `Remove the tables` to `Remove all tables`

File: README.md [hunk 2]
```diff
@@ -41,3 +41,7 @@ If you would like to follow along in the series - The playlist link for this vid
...
+- Make your changes to the database then run the command again (`php artisan db:seed`) to confirm the changes.
```
Comment (tthompson899) [inline README.md]: Request:
Change: Make your changes to the database then run the command again (`php artisan db:seed`) to confirm the changes.

To be:
If you need to make changes to the database:
- Run the command: `php artisan migrate:fresh --seed`
- Make your changes
- Then run the command `php artisan db:seed` and confirm your changes in the database.

**Review by tthompson899 — CHANGES_REQUESTED** [review]
@LuluNwenyi Overall, really great work! Thanks for doing this so quickly.
I have a couple changes I found as I was reading it - None of which you did incorrectly, I just want to make sure it's fully understood to the user how to run the tests.

File: README.md (hunk 2 above)
Comment (LuluNwenyi) [reply README.md]: Thanks for your feedback, Tiffany. I've committed the changes for you to review.

File: README.md [hunk 3]
```diff
@@ -41,3 +41,11 @@ If you would like to follow along in the series - The playlist link for this vid
 Use Postman [install here](https://www.postman.com/downloads/) or browser to visit api route
     - Example route to visit [Get all users](http://appointment.test/api/users)
 
+## How to run Test
+- Initialise the database seeder by running the command: `php artisan db:seed`
+- Remove all tables and add new data: `php artisan migrate:fresh --seed`
+
+**If you need to make changes to the database;**
```
Comment (tthompson899) [inline README.md]: Really close with this! change the semicolon(;) to a colon(:)

**Review by tthompson899 — CHANGES_REQUESTED** [review]
@LuluNwenyi 1 last small change. Almost there 😃

File: README.md (hunk 3 above)
Comment (LuluNwenyi) [reply README.md]: Oh alright

Comment (LuluNwenyi) [reply README.md]: Done @tthompson899

**Review by tthompson899 — APPROVED** [review]
Thanks so much for doing this! 😃

**Pull request merged**